integrations/SnapPixel/util.js

```diff
@@ -14,21 +14,25 @@ const sendEvent = (event, payload) => {
   }
 };
 
-const eventPayload = (message) => {
+const commonEventPayload = (message) => {
```

Comment:
This method name would be more appropriate.
```suggestion
const getCommonEventPayload = (message) => {
```

Reply:
Done